resources/definitions/cubicon_common.def.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": 2,
+    "version": 3,
```

Comment:
That isn't right. The version here is about the schema version. It needs to be 2...

Reply:
I think you misunderstood what I was trying to say. The changes that you made require a script to be written that handles upgrades. You can find examples of those scripts here: https://github.com/Ultimaker/Cura/tree/main/plugins/VersionUpgrade

Reply:
I'll change the schema version back to 2.
There are many versions of the plugin version upgrade, which version should it be?

Reply:
It's not the definition that needs to change, it's the extruder that need to change. An example of a change like that can be found in https://github.com/Ultimaker/Cura/blob/main/plugins/VersionUpgrade/VersionUpgrade41to42/VersionUpgrade41to42.py

Reply:
I understand that VersionUpgrade is required to change the profile entry.
But I don't know how to do VersionUpgrade.
Is there a guide to refer to?